rust/azure_iot_operations_services/src/azure_device_registry/models/device.rs

```diff
@@ -49,19 +49,19 @@ pub struct Device {
 #[derive(Debug, Clone)]
 /// Represents a discovered device in the Azure Device Registry service.
 pub struct DiscoveredDevice {
-    /// The 'attributes' Field.
+    /// A set of key-value pairs that contain custom attributes set by the customer.
     pub attributes: HashMap<String, String>, // if empty hashmap, we can represent as None on generated model
-    /// The 'endpoints' Field.
+    /// Connection endpoint URL a device can use to connect to a service.
```

Comment:
this feels like the wrong description for this field. @abhipsaMisra is this accurate? (Oliva, don't hold the PR on this comment)